### PR TITLE
add flip and autopep8

### PR DESCRIPTION
**Pull request by Gustry**
## Description

@jef-n These packages are required to launch the prepare-commit script.

## Checklist

- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `Fixes #11111` at the bottom of the commit message
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#contributing-to-qgis) before each commit
- [x] I have evaluated whether it is appropriate for this PR to be backported, backport requests are left as label or comment

### CODE REVIEW

**Comment by m-kuhn** [comment]
Hmm, I think they are development dependencies, not build dependencies. Or is committing a required action for building?

**Comment by Gustry** [comment]
Yes, it's development dependencies. So should I just update the readme?

**Comment by m-kuhn** [comment]
I think yes, that's a documentation issue (and/or the script itself could give a meaningful error if the tools are not installed)

**Comment by Gustry** [comment]
Ok, I can update the INSTALL file. There is also `git` in this list so I was wondering.
https://github.com/qgis/QGIS/blob/master/debian/control.in#L75

**Comment by m-kuhn** [comment]
Possibly to get the sha for the about dialog. Not sure.

File: scripts/astyle.sh [hunk 1]
```diff
@@ -53,6 +53,8 @@ if ! type -p flip >/dev/null; then
 		}
 	else
 		echo "flip not found" >&2
+		echo "on debian, try:"
+		echo "apt install flip"
```
Comment (m-kuhn) [inline scripts/astyle.sh]: How does that look?

```suggestion
echo "Try:"
if [[ -f /etc/fedora-release ]]; then
    echo "  dnf install dos2unix";
elif [[ -f debian_version ]]; then
    echo "  apt install flip";
else
    echo "  installing flip or dos2unix from your package manager";
fi
```

Comment (Gustry) [reply scripts/astyle.sh]: Love it of course ;-)

**Review by m-kuhn — APPROVED** [review]
:heart:

**Pull request merged**